### DISCUSSION TITLE
update templates

### DISCUSSION THREAD

**Discussion by marvinmoran** [primefaces/community#2299] [PrimeFaces Templates] [answered]
Hello, I have several questions about the templates. Currently I have acquired the license for some templates, for example Rain, Rome, which I already use in some projects in production. As I see, they no longer appear in the store. The question is, will you no longer see support for those templates?  I also see that the current templates have not been improved while the react and vue templates have many improvements. At some point will they update all the templates to better versions?

**Comment by melloware** [comment, marked as answer]
@marvinmoran PrimeTek posted about this exact topic: https://github.com/orgs/primefaces/discussions/1806

I have Rain and updated it to PF14, updating your themes for yourself to latest PF is not that difficult.  Shoot me a DM on Discord if you need any help.

**Comment by marvinmoran** [comment]
Thank you very much, I have already updated it to Primefaces 14 and Jakarta 10, I would like you to release new more modern themes since my project works very well, but I want to improve the appearance and design, my question is if this year you will release new templates or update the that were left in the store

**Reply by melloware** [reply]
When you say "you" I don't work for PrimeTek....

**Comment by marvinmoran** [comment]
I was referring to the fact that I already updated the rain and roma template, which are the ones I have already purchased.

**Reply by melloware** [reply]
Isn't this a duplicate of what you already asked: https://github.com/orgs/primefaces/discussions/1282 ???

**Comment by marvinmoran** [comment]
well, I think so, what happens is that they didn't answer me, it is important to make decisions if I continue with primefaces or start using other technologies

**Reply by melloware** [reply]
I suggest you contact PrimeTek directly through your PRO account. They don't monitor these forums regularly is more for community to support the community.
